docs/architecture/solana/migration-faq.mdx

```diff
@@ -23,7 +23,8 @@ If a Wallet displays 1 SOL, the app may be on the `devnet` instead of the `mainn
 Wallet, be sure to switch to `mainnet-beta` in settings if it's not already.
 
 If the Wallet still shows 1 SOL while `mainnet-beta` is selected, this display error may resolve by
-switching to `devnet` and back to `mainnet-beta`, or updating the app.
+switching to `devnet` and back to `mainnet-beta`, updating the app or choosing different RPC from 
```

Comment:
The wallet does not support choosing a different rpc

Reply:
@JustAnotherDevv 
The section about the 1 SOL issue can be removed. This was fixed in a later version of the wallet

Reply:
> The wallet does not support choosing a different rpc

My bad, I thought it was referring to generic Solana wallets that can use different RPCs.

Reply:
> @JustAnotherDevv The section about the 1 SOL issue can be removed. This was fixed in a later version of the wallet

Done, removed the section now instead of changing it.